shaded/client/pom.xml

```diff
@@ -261,6 +261,10 @@
                   <pattern>javassist</pattern>
                   <shadedPattern>${shading.prefix}.javassist</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>software/amazon/ion/</pattern>
```

Comment:
alphabetical order?  I think probably it might be better if we could make the relocation entries in alphabetical order, what do you think?

Reply:
You're right. I just submitted a commit to update it.